### ISSUE TITLE
ModuleNotFoundError: No module named 'novauniverse.endpoints'

### ISSUE THREAD

**Issue by THEGOLDENPRO** [NovaUniverse/NovaUniverse.py#5]
```
Traceback (most recent call last):
  File "c:/Users/{USER}/Desktop/dsdfd.py", line 1, in <module>
    import novauniverse as nova
  File "C:\Users\{USER}\AppData\Local\Programs\Python\Python38\lib\site-packages\novauniverse\__init__.py", line 1, in <module>       
    from ._find_ import Player, Session
  File "C:\Users\{USER}\AppData\Local\Programs\Python\Python38\lib\site-packages\novauniverse\_find_.py", line 1, in <module>
    from .endpoints import nova, mojang
ModuleNotFoundError: No module named 'novauniverse.endpoints'
```

**Comment by THEGOLDENPRO** [comment]
**I'm aware of this bug and I am working on a fix as we speak.**

**Comment by THEGOLDENPRO** [comment]
Fixed in ``v1.0a5``: https://pypi.org/project/novauniverse/1.0a5/
